backend/migrate.sh

```diff
@@ -3,56 +3,97 @@
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 export PYTHONPATH=$SCRIPT_DIR:$PYTHONPATH
 
-# Installing dependencies using Poetry
-echo "Installing dependencies with Poetry..."
-poetry install
+echo '   __   _____________  ___  ___    __  ____               __  _         '
+echo '  / /  / ___/ __/ __/ / _ \/ _ )  /  |/  (_)__ ________ _/ /_(_)__  ___ '
+echo ' / /__/ /__/ _/_\ \  / // / _  | / /|_/ / / _ `/ __/ _ `/ __/ / _ \/ _ \'
+echo '/____/\___/_/ /___/ /____/____/ /_/  /_/_/\_, /_/  \_,_/\__/_/\___/_//_/'
+echo '                                         /___/                          '
+echo -e
+
+# Function to seed the database
+seed_database() {
+    echo "🌱  Seeding the database..."
```

Comment:
This looks good, please just add a comment that calling seed_database will default to using 'dev' seeds for the development environment.

Reply:
Added the comment. Thanks, Alex, for the code review!